lampo-common/src/conf.rs

```diff
@@ -133,6 +145,16 @@ impl LampoConf {
 
         Ok(conf)
     }
+
+    // These functions should be called when we get something like
+    // liquidity=consumer of liquidity=provider inside lampo.conf
+    pub fn configure_as_liquidity_consumer(&mut self) {
+        self.liquidity = Some("Consumer".to_string())
```

Comment:
```suggestion
        self.liquidity = Some("Consumer".to_string())
```
use an enum like `LiquisityProvider::Consumer`